Subject: [PATCH] feat(verification): add VerificationCode
---
Index: backend/codesirius/authentication/admin.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/backend/codesirius/authentication/admin.py b/backend/codesirius/authentication/admin.py
--- a/backend/codesirius/authentication/admin.py	(revision 098f7990e896e9baf1c37a9d02a1355b8f72acc0)
+++ b/backend/codesirius/authentication/admin.py	(date 1738351026239)
@@ -2,6 +2,8 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
 
+from authentication.models import VerificationCode
+
 User = get_user_model()
 
 
